tests/aiohttp/test_graphql_transport_ws.py

```diff
@@ -285,6 +285,55 @@ async def test_duplicated_operation_ids(aiohttp_client):
         assert data.extra == "Subscriber for sub1 already exists"
 
 
+async def test_reused_operation_ids(aiohttp_client):
```

Comment:
As these are fairly complicated tests, could you add some docstrings describing what functionality is being tested

Reply:
Added docstrings.
It is a bit annoying that we have three identical sets of tests, testing _internal_ functionality, that is in theory quite independent of the integration in use.   Would it make sense to separate "integration" tests from unit tests?

Reply:
cc @patrick91, I think we were discussing ideas for doing this last week

Reply:
@kristjanvalur totally, I've opened up an issue for this: https://github.com/strawberry-graphql/strawberry/issues/1809